string ObjID
geometry_msgs/PoseStamped pose